batavia/core/builtins.js

```diff
@@ -1026,7 +1035,7 @@ batavia.builtins.set = function(args,kwargs) {
     if (args && args.length > 1) {
         throw new batavia.builtins.TypeError("set expected at most 1 arguments, got " + args.length);
     }
-	return new batavia.types.Set(args[0]);
+	  return new batavia.types.Set(args[0]);
```

Comment:
Looks like a tab/spaces problem here - although possibly one that existed previously.